.npmignore

```diff
@@ -0,0 +1 @@
+package-lock.json
```

Comment:
this file should be commited, please take a look: https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md